Packs/Censys/Integrations/Censys/Censys.py

```diff
@@ -51,14 +51,22 @@ def censys_search_command():
     args = demisto.args()
     query = args.get('query')
     index = args.get('index')
-
+    results = []
     url_suffix = 'search/{0}'.format(index)
     data = {
         "query": query,
         "page": 1
     }
     raw = send_request('POST', url_suffix, json.dumps(data))
-    readable = tableToMarkdown("Search results for {0} in {1}".format(query, index), raw["results"])
+    results += raw["results"]
+    if 'metadata' in raw:
+        if 'pages' in raw['metadata']:
+            for x in range(int(raw['metadata']['pages'])):
+                data["page"] += 1
+                raw = send_request('POST', url_suffix, json.dumps(data))
+                results += raw["results"]
```

Comment:
Please read the following article about Pagination: [Pagination in integration commands](https://xsoar.pan.dev/docs/integrations/code-conventions#pagination-in-integration-commands)
This is the correct way to handle api calls that have pages.

Note: It is important to limit the ```for``` loop iterations (by limiting the number of pages in this case), doing so will prevent the command from running too long or crushing due to memory error when there are many pages.